### ISSUE TITLE
Create method to PATCH existing issue with failed moderation template

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#3141]
Using the GitHub API, we send a PATCH request to the existing public issue and replace the body with the “this is unacceptable” template. Also:

1. Add labels from private issue. Add an “status-moderation-rejected” label.
2. Close the public issue as WONTFIX.

**Comment by miketaylr** [comment]
This is probably a duplicate of #3140?

Depends on #3150.

**Comment by karlcow** [comment]
> Add labels from private issue.

do we need those?

**Comment by karlcow** [comment]
See https://github.com/webcompat/webcompat.com/pull/3167/commits/e4aa9d22eae8ab99541c659b7e5c0fccbba0c0b1
